gather_cnv_network

```diff
@@ -18,7 +18,6 @@ do
     for pod in $NS_PODS
     do
         echo "POD: $pod"
-        /usr/bin/openshift-must-gather inspect -n $ns pod/${pod}
```

Comment:
Why we stop using inspect on for the pods? Are we should we are not loosing information here?

Reply:
The pod manifests are gathered during namespace gathering: "must-gather inspect ns/*"
After grepping through the results, I see that the logs too are gathered, which makes the network script void.
Moving the ns list to the general list, dropping the file completely.